setup.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from setuptools import setup, find_packages
+
+with open('README.rst', 'rb') as f:
+    LONG_DESCRIPTION = f.read().decode('utf8')
+
+# Single source the version.
+version_file = os.path.realpath(os.path.join(
+    os.path.dirname(__file__), 'pymongoexplain', 'version.py'))
+version = {}
+with open(version_file) as fp:
+    exec(fp.read(), version)
+
+setup(
+    name="pymongoexplain",
+    version=version['__version__'],
+    description="Explainable CRUD API for PyMongo",
+    long_description=LONG_DESCRIPTION,
+    packages=find_packages(exclude=['test']),
+    author="Julius Park",
+    url="https://github.com/mongodb-labs/pymongoexplain",
+    keywords=["mongo", "mongodb", "pymongo"],
+    test_suite="test",
```

Comment:
You'll need to add this so that the setup script will automatically download pymongo when running `setup.py test`:
```
    tests_require=["pymongo"],
```

Reply:
Done.

Reply:
Actually, you need to add `pymongo` to `install_requires` since this package itself depends on PyMongo (not just the tests). See https://packaging.python.org/guides/distributing-packages-using-setuptools/#install-requires for details.
You also need to determine what minimum version of PyMongo you will be needed. I think to keep things simple, we can go with 3.10.1 (the last stable release) for now.

Reply:
Done.